bot/kodiak/pull_request.py

```diff
@@ -138,6 +152,7 @@ def __init__(
         number: int,
         dequeue_callback: Callable[[], Awaitable],
         queue_for_merge_callback: Callable[[], Awaitable[Optional[int]]],
+        client: Optional[Type[Client]] = None,
```

Comment:
not sure if this would work but:

```suggestion
        client: Type[Client] = Client,
```

and remove the `or Client` in the body of the method